app/models/miq_worker/runner.rb

```diff
@@ -386,7 +385,8 @@ def heartbeat_to_drb
   end
 
   def heartbeat_to_file
-    FileUtils.touch(@worker.heartbeat_file)
+    timeout = worker_settings[:heartbeat_timeout] || Workers::MiqDefaults.heartbeat_timeout
```

Comment:
Sorry for the delayed reply on this, but yeah... hrmmm... not sure how we would be able to do this... would have to ponder for a bit...

I assume you are referring to this bit in particular:  https://github.com/ManageIQ/manageiq/blob/8b04e133/app/models/miq_server/worker_management/monitor/settings.rb#L38-L41

That said, depending on the queue changes, there might not be available if the `current_timeout` stops being a thing in the new architecture:  https://github.com/ManageIQ/manageiq/blob/8b04e133/app/models/miq_worker.rb#L451-L454